community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/cleanup_compute/variables.tf

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (C) SchedMD LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "project_id" {
+  type        = string
+  description = "Project ID"
+}
+
+
+variable "slurm_cluster_name" {
+  type        = string
+  description = "Name of the Slurm cluster"
+}
+
+variable "enable_cleanup_compute" {
+  description = <<EOD
+Enables automatic cleanup of compute nodes and resource policies (e.g.
+placement groups) managed by this module, when cluster is destroyed.
+
+*WARNING*: Toggling this off will impact the running workload.
+Deployed compute nodes will be destroyed.
+EOD
+  type        = bool
+}
+
+variable "universe_domain" {
```

Comment:
It seems like we either need default values for these API configurations here or in the bash script. Am I not seeing where the default/simple case (GA API, standard universe) is being set?

Reply:
It's in `community/modules/scheduler/schedmd-slurm-gcp-v6-controller/variables.tf`
I would prefer to have at most one place were defaults are defined, and touching controller module interface increases scope needlessly.

Reply:
I agree, I just couldn't find it via GitHub UI easily. I also think this is a case where the sub-module is an internal implementation detail so exposing the default at the top-level is the right thing.